lib/src/components/NxVulnerabilityIndicatorLegend/NxVulnerabilityIndicatorLegend.scss

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@use '../../scss-shared/nx-container-helpers';
+@use '../../scss-shared/nx-text-helpers';
+@use '../../scss-shared/nx-dark-mode-helpers';
+
+.nx-vulnerability-indicator-legend {
+  $max-grid-columns: 6;
```

Comment:
Shouldn't this be 5 now

Reply:
> Shouldn't this be 5 now

Ah yes: [64ca095](https://github.com/sonatype/sonatype-react-shared-components/pull/1199/commits/64ca09587889eadcb265b0b8d95ab40526f14c77)